docs/modules/ROOT/pages/index.adoc

```diff
@@ -7,3 +7,59 @@ This gem introduces two core classes of RuboCop:
 
 See xref:node_pattern.adoc["Node Pattern"] to get yourself familiar with ``NodePattern``'s
 capabilities.
+
+This gem is essentially an extension of the https://github.com/whitequark/parser/[`parser`] gem, which adds a simpler and more powerful object-oriented API to make it easier to work with nodes.
```

Comment:
It might be difficult to convey, but `rubocop-ast` is really two things: a wrapper for `parser`'s `AST::Node` *and* `NodePattern`. They could even be separated... Could you tweak the wording here?

Reply:
Also, a big change that might be worth mentionning (either here or in the examples) is that there we provide `parent`, contrary to `parser`...